### PR TITLE
[New Extension] RayDocs

### PR DESCRIPTION
**Pull request by teziovsky**
## Description

Search Raycast's documentation quickly, copy the URL, or open it in your browser! 🔥

## Screencast

raydocs-1

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/raydocs/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { List } from "@raycast/api";
+import { useState } from "react";
+import LinkItem from "./components/LinkItem";
+import { Link } from "./types";
+import { scrapDocs } from "./utils";
```
Comment (andreaselia) [inline extensions/raydocs/src/index.tsx:5]: Should `scrapDocs` be `scrapeDocs`?

Comment (teziovsky) [reply extensions/raydocs/src/index.tsx:5]: It could be getLinks or something simmilar 🤔

Comment (andreaselia) [reply extensions/raydocs/src/index.tsx:5]: Sounds good, getLinks, scrapeMenuLinks, scrapeLinks, fetchLinks, fetchMenuLinks, etc. could all be decent options 😁

Comment (teziovsky) [reply extensions/raydocs/src/index.tsx]: Done 🧑‍🚒

**Comment by teziovsky** [comment]
> Hey @teziovsky 👋
> 
> I just took this for a spin and everything looks good aside from when opening the command, you are greeted with a "No Results" message until the links load.
> 
> I think your `isLoading` state needs to be `true` by default, as I can see you have code already to set it to `false`?

Yes, you're correct! I've already fixed it 😎

**Comment by andreaselia** [comment]
Awesome, super fast ⚡  Thanks for sorting that @teziovsky, it looks good to me now. Just need to await a member of the Raycast team to review and merge if they're happy and then it should be on the store in no time at all 😁

**Review by andreaselia — APPROVED** [review]

**Review by pernielsentikaer — APPROVED** [review]
Hello 👋

I gave it a quick spin, works as expected, approved 🔥

**Pull request merged**